packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
+
 ## 2.1.11
 
+* Fixes violations of new analysis option use_named_constants.
```

Comment:
This got merged incorrectly; you added this to the release notes of an already-released version, and didn't add a new version update to release the changed production code. (Normally CI would have caught this, but the override we added for video_player masked it).

Could you do a follow-up to move this line to a new 2.1.12?

Reply:
I'll send that up shortly

Reply:
new PR here https://github.com/flutter/plugins/pull/6289